.goreleaser.yml

```diff
@@ -1,8 +1,10 @@
 project_name: influx-cli
 
-# Do not make github release
+# Do not make github release or generate a changelog
 release:
   disable: true
+changelog:
```

Comment:
Not 100% necessary, but might be helpful to prevent confusion due to the generate goreleaser CHANGELOG, which is not relevant since we are generating our own.